geocoding/geocode.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import pathlib
+
+import fire
+import googlemaps
+import pandas as pd
+from tqdm import tqdm
+
+logging.basicConfig(
+    filename="geocoding.log",
+    filemode="a",
+    format="%(asctime)s,%(msecs)d %(name)s %(levelname)s %(message)s",
+    datefmt="%H:%M:%S",
+    level=logging.INFO,
+)
+
+logging.info("Logger Setup Complete")
+
+
+def make_dir():
+    """Responsible for creating the folder in which the CSV's will
+    be stored in.
+    """
+    sheets_dir = pathlib.Path.cwd() / "Sheets"
+
+    if not sheets_dir.is_dir():
+        sheets_dir.mkdir()
```

Comment:
You can do this inline in 2 lines:

```suggestion
from pathlib import Path
sheets_dir:Path = Path.cwd() / "Sheets"
sheets_dir.mkdir(parents=True, exist_ok=True)
```

Or even one line:
```python
(Path.cwd() / "Sheets").mkdir(parents=True, exist_ok=True)
```

Reply:
Removed the overly elaborate function. Replaced with - 
`(Path.cwd() / "Sheets").mkdir(parents=True, exist_ok=True)`